modules/ROOT/partials/configuration/mergetags_list.adoc

```diff
@@ -0,0 +1,74 @@
+[[mergetags_list]]
+== `+mergetags_list+`
+
+`+mergetags_list+` is an object array that specifies the menu content used for merge tags insertion. Every object specifies the configuration of a submenu or a menu item.
+
+NOTE: If the `+mergetags_list+` option is not set, or contains no entries, both the merge tags toolbar button and the merge tags menu item are hidden. Merge tag autosuggestions are also disabled if the `+mergetags_list+` option is not set, or contains no entries.
+
+*Type:* `+Array+`
+
+*Default value:* `+undefined+`
+
+=== Menu item properties
+
+[cols="1,1,1,4",options="header"]
+|===
+|Name|Type|Required|Description
+|title |string |optional |If set: the menu item label to display instead of the `+value+`
+|value |string |required |The merge tag content to be inserted after the menu item is activated
+|===
+
+=== Submenu properties
+
+[cols="1,1,1,4",options="header"]
+|===
+|Name|Type|Required|Description
+|title |string |required |The submenu label to display
+|menu |Array |required |The list of nested submenus and menu items
+|===
+
+=== Example: Using `+mergetags_list+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'mergetags',
+  mergetags_list: [
+    {
+      value: 'Current.Date',
+      title: 'Current date in DD/MM/YYYY format'
+    },
+    {
+      value: 'Current.Time',
+    },
+    {
+      title: 'Person',
+      menu: [
+        {
+          value: 'Person.Name.First',
+          title: 'first name'
+        },
+        {
+          value: 'Person.Name.Last',
+          title: 'last name'
+        },
+        {
+          value: 'Person.Name.Full',
+          title: 'full name'
+        },
+        {
+          title: 'Email',
+          menu: [{
+            value: 'Person.Email.Work'
+          },
+          {
+            value: 'Person.Email.Home'
+          },
+          ]
```

Comment:
This formatting doesn't follow code style conventions and is inconsistent even within this example.
```suggestion
          menu: [
            {
              value: 'Person.Email.Work'
            },
            {
              value: 'Person.Email.Home'
            }
          ]
```